pkg/genericapiserver/options/server_run_options.go

```diff
@@ -76,6 +76,7 @@ type ServerRunOptions struct {
 	TargetRAMMB            int
 	TLSCAFile              string
 	WatchCacheSizes        []string
+	EnableCURLDebug        bool
```

Comment:
This doesn't belong as generic option.  Not all API servers even have etcd.  Find the `etcdOptions` to add it there.